astropy/table/tests/test_mixin.py

```diff
@@ -100,16 +101,32 @@ def test_io_ascii_write():
         if fmt['Write'] and '.fast_' not in fmt['Format']:
             out = StringIO()
             t.write(out, format=fmt['Format'])
-
+            
+def test_io_Quantity_write():
+    """
+    Test that table with Quantity mixin column can be written by io.votable,
+    io.fits, and io.misc.hdf5. No validation of the output is done,
+    this just confirms no exceptions.
+    """
+    t = QTable()
+    t['a'] = Quantity([1,2,4], unit='Angstrom')
+    for fmt in ('fits', 'votable', 'hdf5'):
+        if fmt == 'hdf5' and not HAS_H5PY:
+            continue
+        out = StringIO()
+        t.write(out, format=fmt)
```

Comment:
Do all these formats round-trip?  I think at least FITS should, so where possible read the `out` handle back in the same format with `QTable.read(...)` and make sure it is the same table (in particular that the unit made it through the round-trip).